cmd/nerdctl/images.go

```diff
@@ -166,6 +175,12 @@ func printImages(ctx context.Context, clicontext *cli.Context, client *container
 				return err
 			}
 		} else {
+			if repository == "" {
+				repository = "<none>"
+			}
+			if tag == "" {
+				tag = "<none>"
+			}
```

Comment:
I think it s better to keep them empty. otherwise you have to adapt the test https://github.com/containerd/nerdctl/blob/e0f067e3f657e68b77e8db823b28842659b86150/pkg/imgutil/imgutil_test.go#L25